sdk/tables/azure-data-tables/inc/azure/data/tables/account_sas_builder.hpp

```diff
@@ -202,24 +178,12 @@ namespace Azure { namespace Data { namespace Tables { namespace Sas {
     All = ~0,
   };
 
-  inline AccountSasPermissions operator|(AccountSasPermissions lhs, AccountSasPermissions rhs)
-  {
-    using type = std::underlying_type_t<AccountSasPermissions>;
-    return static_cast<AccountSasPermissions>(static_cast<type>(lhs) | static_cast<type>(rhs));
-  }
-
-  inline AccountSasPermissions operator&(AccountSasPermissions lhs, AccountSasPermissions rhs)
-  {
-    using type = std::underlying_type_t<AccountSasPermissions>;
-    return static_cast<AccountSasPermissions>(static_cast<type>(lhs) & static_cast<type>(rhs));
-  }
-
   /**
    * @brief AccountSasBuilder is used to generate an account level Shared Access Signature
    * (SAS) for Azure Storage services.
    */
-  struct AccountSasBuilder final
-  {
+  class AccountSasBuilder final {

```

Comment:
It is strange that clang-format doesn't complain about this curly brace being on the same line
```suggestion
  class AccountSasBuilder final
  {
```

Reply:
keeps formatting it like that , even if i put it like suggested the formatting pulls it back to the previous version